pulsar-client-api/src/main/java/org/apache/pulsar/client/api/schema/GenericRecord.java

```diff
@@ -32,9 +33,11 @@
     /**
      * Return schema version.
      *
-     * @return schema version.
+     * @return schema version, or null if the information is not available.
      */
-    byte[] getSchemaVersion();
+    default byte[] getSchemaVersion() {
```

Comment:
why do you need to change the default implementation?

Reply:
As I am touching this interface I would like to add this default implementation as well.

When you are coding your own implementation of GenericRecord, for instance in order to Produce message, it is awkward to deal with this method, as you do not know what to return.

I can revert this change from this patch and follow up with a separate discussion. not problem

Reply:
@sijie 
I have reverted the default implementation of getSchemaVersion()